languages/python310/compile

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+COMPILE_DIR="$(dirname "$0")"
+
+cp $COMPILE_DIR/../python3/* $COMPILE_DIR/
+
+sed -i "s/python-3.8-embed/python-3.10-embed/g" $COMPILE_DIR/main.go
```

Comment:
Feels a little dirty

Reply:
Yeah... if you build it with `nix build .#prybar-python310` it's cool because the builds in a separate tmp dir so those files don't end up anywhere. But if you build it with `make prybar-python310` those files do get created. I could `.gitignore` those generated file in the make case. 

Or I could simply duplicate those files.